src/core_modules/capture-core/components/D2Form/actions/form.actions.js

```diff
@@ -5,6 +5,6 @@ export const actionTypes = {
     ADD_FORM_DATA: 'AddFormData',
 };
 
-export function addFormData(formId: string, formValues: Object) {
+export function addFormData(formId: string, formValues: $Shape<{||}> = {}) {
```

Comment:
That is true. I was playing around with this and forgot to return it back to its original shape. Will do now.

Reply:
Flow is a weird thing. This should throw an error since as you say I am explicitly saying here that the only shape of the form values that can be given is `{}`. However no erros being thrown when this function is used with somthing else than `{}`. 

Can you explain why maybe? Just curious here.

Reply:
If I add an object literal with a property as an argument to the caller (and keep the $Shape<{||}), Flow will complain. But yeah, absolutelty agree that Flow is hard to get sometimes.